src/Components/Horoscope/Horoscope.js

```diff
@@ -1,66 +1,38 @@
+    const { day, isLoading, errorMsg, horoscope, image, alt, sign } = props.horoscope;
 
-    componentDidMount() {
-        fetchHoroscope('https://aztro.sameerkumar.website/?sign=aries&day=today')
-        .then(result => {
-            if (typeof result === 'string') {
-                this.setState({
-                  isLoading: false,
-                  errorMsg: result
-                })
-              } else {
-                this.setState({
-                    horoscope: result,
-                    isLoading: false
-                  })
-              }
-        })
-    }
-
-    render() {
-        const { src, alt, signTitle } = this.props.location.state;
-        const { isLoading, errorMsg, horoscope } = this.state;
-        console.log(horoscope)
-        return (
-            <main className="horoscope-main">
-                <img className='horoscope-sign-img' src={src} alt={alt} />
-                <div className='details'>
-                    <h2 className='horoscope-sign-title'>{signTitle}</h2>
-                    {errorMsg && <p className='error-message'>{errorMsg}</p>}
-                    {isLoading && <p className='loading-message'>Loading...</p>}
-                    {horoscope  && 
-                    <section>
-                        <ul>
-                            <li className='days-date'>Current Date: {horoscope.current_date}</li>
-                            <li className='date-range'>Birthday Range: {horoscope.date_range}</li>
-                            <li className='days-color'>Color: {horoscope.color}</li>
-                            <li className='days-compatibility'>Compatibility: {horoscope.compatibility}</li>
-                            <li className='days-description'>Description: {horoscope.description}</li>
-                            <li className='days-lucky-numbers'>Lucky Number: {horoscope.lucky_number}</li>
-                            <li className='days-lucky-time'>Lucky Time: {horoscope.lucky_time}</li>
-                            <li className='days-mood'>Mood: {horoscope.mood}</li>
-                        </ul>
-                        <button className='yesterday'>Yesterday</button>
-                        <button className='today'>Today</button>
-                        <button className='tomorrow'>Tomorrow</button>
-                    </section>
-                    }     
-                </div>
-            </main>
-        )
-    }
+    return (
+        <main className="horoscope-main">
+            {errorMsg && <p className='error-message'>{errorMsg}</p>}
+            {isLoading && <p className='loading-message'>Loading...</p>}
+            {horoscope  &&   
+                <section className="horoscope-content">
+                    <img className='horoscope-sign-img' src={image} alt={alt} />
+                    <div className='details'>
+                        <h2 className='horoscope-sign-title'>{sign.toUpperCase()}</h2>
+                            <ul>
```

Comment:
I might break these lines out a bit more to make it more readable. Just by popping the h3 tags onto their own line I think would help a lot!